docs/data/toolpad/core/introduction/installation.md

```diff
@@ -4,47 +4,47 @@ title: Toolpad Core - Installation
 
 # Installation
 
-## Manual Installation
+<p class="description">Learn how to install Toolpad Core in your local environment.</p>
+
+## Manual installation
 
 Use your preferred package manager to install `@toolpad/core` in your project:
 
 <codeblock storageKey="package-manager">
 
 ```bash npm
-npm install -S @toolpad/core
-```
-
-```bash yarn
-yarn add @toolpad/core
+npm install @toolpad/core
 ```
 
 ```bash pnpm
 pnpm add @toolpad/core
 ```
 
+```bash yarn
+yarn add @toolpad/core
+```
+
 </codeblock>
 
 The Toolpad Core package has a peer dependency on `@mui/material` and `@mui/icons-material`. If you aren't using these already in your project, you can install them with:
 
 <codeblock storageKey="package-manager">
 
 ```bash npm
-npm install -S @mui/material @mui/icons-material @emotion/react @emotion/styled
```

Comment:
Remove the `-S` isn't this the default https://docs.npmjs.com/cli/v10/commands/npm-install#save?